### DISCUSSION TITLE
Contributing to this project

### DISCUSSION THREAD

**Discussion by OpenSourceSimon** [is-a-dev/register#3920] [Q&A] [answered]
Hey, I saw this project and it looks good. I don't need a domain for myself, but I would love to approve and merge pull requests to help other people. I want to help with this project, unfortunately I'm a Python developer and this isn't Python. How can I help?

**Comment by wdhdev** [comment, marked as answer]
Sorry, helpers and maintainers are hand selected. But you could help by joining our Discord Server and helping people there.

**Reply by MaskDuck** [reply]
actually btw i'm a maintainer and i only know python /shrug
